src/Apps/Collect/Components/Filters/index.tsx

```diff
@@ -0,0 +1,7 @@
+export { FilterContainer } from "./FilterContainer"
+
+export { MediumFilter } from "./MediumFilter"
+export { PriceRangeFilter } from "./PriceRangeFilter"
+export { SortFilter } from "./SortFilter"
+export { TimePeriodFilter } from "./TimePeriodFilter"
+export { WaysToBuyFilter } from "./WaysToBuyFilter"
```

Comment:
So that the context renderProps pattern doesn't feel too verbose, could create a factory function that wraps and injects items in as props:

```jsx
export const MediumFilter = makeFilter(...)
```